code/datums/components/archaeology.dm

```diff
@@ -63,7 +63,7 @@
 					if(isplatingturf(OT))
 						var/turf/open/floor/plating/POT = parent
 						POT.icon_plating = "[POT.postdig_icon]"
-					OT.icon_state = "[OT.postdig_icon]"
+						POT.icon_state = "[OT.postdig_icon]"
```

Comment:
I thought this was initially the problem then I realized I was dumb and didn't fix it, although this also works